### PR TITLE
With owner

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: pkg/helpers/apply/apply_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+			applier := applierBuilder.
+				WithClient(kubeClient, apiExtensionsClient, dynamicClient).
+				WithTemplateFuncMap(FuncMap()).WithOwner(managedClusterOwner, false, false, scheme.Scheme).
+				Build()
+
+			_, err := applier.ApplyCustomResources(reader, nil, false, "", "ownerref/managedcluster.yaml")
+			Expect(err).To(BeNil())
+		})
+		By("Checking Ownerref", func() {
+			mc, err := clusterClient.ClusterV1().ManagedClusters().Get(context.TODO(), "my-managedcluster", metav1.GetOptions{})
```
Comment (qiujian16) [inline pkg/helpers/apply/apply_suite_test.go:142]: This seems to add a onwer of one mc to another?

Comment (itdove) [reply pkg/helpers/apply/apply_suite_test.go:142]: yes it is just and example, I used 2 mc, could be one mc and something else. it doesn't matter which CRD I use... it is just to test that one resource becomes the owner of another.

**Pull request merged**